api-tests/dev_apis/crypto/test_c073/test_c073.c

```diff
@@ -139,8 +139,7 @@ int32_t psa_pake_set_context_test(caller_security_t caller __UNUSED)
    /* Expect a bad state if psa_pake_set_context is called on aborted inactive operation object */
         status = val->crypto_function(VAL_CRYPTO_PAKE_SET_CONTEXT,
                                       &operation,
-                                      check1[i].context,
-                                      check1[i].context_len);
+                                      (const uint8_t*)"context", 7);
```

Comment:
To maintain consistency with other test cases, check for bad state can be added as
```
if( check1[i].expected_status == PSA_SUCCESS)
{
    status = val->crypto_function(VAL_CRYPTO_PAKE_SET_CONTEXT, &operation, check1[i].context, check1[i].context_len);                                           
}